src/components/card/card.ts

```diff
@@ -0,0 +1,69 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import CardScss from './card.scss';
+
+/**
+ * Card.
+ * @fires on-card-click - Captures the click event of clickable card and emits the original event details. Use `e.stopPropogation()` / `e.preventDefault()` for any internal clickable elements when card type is `'clickable'` to stop bubbling / prevent event.
+ * @slot unnamed - Slot for card contents.
+ */
+
+@customElement('kd-card')
+export class Card extends LitElement {
+  static override styles = CardScss;
+
+  /** Card Type. `'normal'` & `'clickable'` */
+  @property({ type: String })
+  type = 'normal';
+
+  /** Card link url for clickable cards. */
+  @property({ type: String })
+  href = '';
+
+  /** Use for Card type `'clickable'`. Defines a relationship between a linked resource and the document. An empty string (default) means no particular relationship. */
+  @property({ type: String })
+  rel = '';
+
+  /** Defines a target attribute for where to load the URL in case of clickable card. Possible options include "_self" (deafult), "_blank", "_parent", "_top" */
+  @property({ type: String })
+  target = '_self';
+
+  /** Hide card border. Useful when clickable card use inside `<kyn-notification>` component. */
+  @property({ type: Boolean })
+  hideBorder = false;
```

Comment:
Maybe we should apply this to the "normal" variant as well, otherwise it's kind of confusing that it does nothing.

Reply:
yes normal variant is there. Story name `Simple`